packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -94,7 +94,7 @@ const { Vow$ } = NetworkShape; // TODO #9611
 /**
  * @private
  * @typedef {{
- *   topicKit: RecorderKit<ComosOrchestrationAccountNotification>;
+ *   topicKit: RecorderKit<ComosOrchestrationAccountNotification> | undefined;
```

Comment:
minor, but it shouldn't the key also be optional?
```suggestion
 *   topicKit?: RecorderKit<ComosOrchestrationAccountNotification>;
```

Reply:
Good question. The actual state object will have the key based on the current implementation. I can change to `return { ...(topicKit && { topicKit }) } ` and adopt an optional key.

Exo state migrations are something coming to mind. We might want to later add `topicKit` to all `OrchAccount` instances, so it could be good to have the key there.

If we choose to remove it from state when undefined, we can do the same for `icqConnection`.

Reply:
oh, I didn't notice this is the state shape. Let's leave it as requiring the key for now.